pkg/frontend/frontend.go

```diff
@@ -245,12 +251,9 @@ func (f *frontend) Run(ctx context.Context, stop <-chan struct{}, done chan<- st
 	}
 
 	f.s = &http.Server{
-		Handler:      middleware.Lowercase(f.setupRouter()),
-		ReadTimeout:  10 * time.Second,
-		WriteTimeout: time.Minute,
-		IdleTimeout:  2 * time.Minute,
-		ErrorLog:     log.New(f.baseLog.Writer(), "", 0),
-		BaseContext:  func(net.Listener) context.Context { return ctx },
+		Handler:     middleware.Lowercase(f.setupRouter()),
+		ErrorLog:    log.New(f.baseLog.Writer(), "", 0),
+		BaseContext: func(net.Listener) context.Context { return ctx },
```

Comment:
what are the default timeouts - are they big enough? once this is removed are we relying on context timeouts?

Reply:
by default there is no timeout.  This was more of a concern to me before we were firewalled; now that we are I think it's OK to drop these.